nix/config.nix

```diff
@@ -50,8 +50,8 @@ in
             hash = "sha256-Bt110wXNOHDuYrOTLmYcYkVD6vmuK6N83sCk+O3QUdI=";
           };
           "17" = {
-            version = "17.6";
-            hash = "sha256-4GMKNgCuonURcVVjJZ7CERzV9DU6SwQOC+gn+UzXqLA=";
+            version = "17.7";
```

Comment:
I think we should make the update to 17.7 in our other PR where you were doing that #1926